authors/apps/authentication/backends.py

```diff
@@ -1,10 +1,57 @@
-# import jwt
-#
-# from django.conf import settings
-#
-# from rest_framework import authentication, exceptions
-#
-# from .models import User
+import jwt
+from django.conf import settings
+from rest_framework import authentication, exceptions
+from .models import User
+from .serializers import LoginSerializer
+from .jwt_generator import jwt_decode
 
-"""Configure JWT Here"""
 
+class JWTAuthentication(authentication.BaseAuthentication):
+    def authenticate(self, request, username=None, password=None):
+        """
+        Returns a `User` if a correct username and password have been supplied
+        using HTTP Basic authentication.  Otherwise returns `None`.
+        """
+
+        auth_token = request.META.get('HTTP_AUTHORIZATION', b'').split()
```

Comment:
Due to the nature of the feature, this might be working, but I'll leave this comment as a reminder that the 'Bearer' parameter is still necessary.